riscv/v_ext_macros.h

```diff
@@ -208,9 +208,9 @@ static inline bool is_aligned(const unsigned val, const unsigned pos)
   require(P.VU.vsew >= e8 && P.VU.vsew <= e64); \
   require_vector(true); \
   reg_t vl = P.VU.vl->read(); \
-  reg_t sew = P.VU.vsew; \
+  reg_t UNUSED sew = P.VU.vsew; \
```

Comment:
I assume these are used by some instructions but not all. What happens if an UNUSED variable is actually used? Is it possible the compiler could emit bad code in that case?

Reply:
Right, `__attribute__((unused))` means the variable _might_ be unused; it doesn't result in undefined behavior if the variable actually is used.